### ISSUE TITLE
Crash When Editing Custom Theme & Florisboard Not Default Input

### ISSUE THREAD

**Issue by whispy** [florisboard/florisboard#741]
#### Short description
When I try to edit a custom theme and Florisboard is not set as the default input, Florisboard crashes

#### Steps to reproduce
1. Set another keyboard as default input
2. Open Florisboard settings
3. Navigate to themes
4. Edit custom theme
5. Crash occurs

I'm happy to provide more info/troubleshooting if needed.

~~~ 1619112811810.stacktrace ~~~

java.lang.RuntimeException: Unable to start activity ComponentInfo{dev.patrickgold.florisboard/dev.patrickgold.florisboard.settings.ThemeManagerActivity}: android.view.InflateException: Binary XML file line #37: Binary XML file line #37: Error inflating class dev.patrickgold.florisboard.ime.text.keyboard.KeyboardView
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2778)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2856)
	at android.app.ActivityThread.-wrap11(Unknown Source:0)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1589)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:164)
	at android.app.ActivityThread.main(ActivityThread.java:6494)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:440)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
Caused by: android.view.InflateException: Binary XML file line #37: Binary XML file line #37: Error inflating class dev.patrickgold.florisboard.ime.text.keyboard.KeyboardView
Caused by: android.view.InflateException: Binary XML file line #37: Error inflating class dev.patrickgold.florisboard.ime.text.keyboard.KeyboardView
Caused by: java.lang.reflect.InvocationTargetException
	at java.lang.reflect.Constructor.newInstance0(Native Method)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:334)
	at android.view.LayoutInflater.createView(LayoutInflater.java:647)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:790)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:730)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:863)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:866)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:866)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:866)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:866)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
	at dev.patrickgold.florisboard.databinding.ThemeManagerActivityBinding.inflate(ThemeManagerActivityBinding.java:101)
	at dev.patrickgold.florisboard.databinding.ThemeManagerActivityBinding.inflate(ThemeManagerActivityBinding.java:95)
	at dev.patrickgold.florisboard.settings.ThemeManagerActivity.onCreateBinding(ThemeManagerActivity.kt:159)
	at dev.patrickgold.florisboard.settings.ThemeManagerActivity.onCreateBinding(ThemeManagerActivity.kt:52)
	at dev.patrickgold.florisboard.ime.core.FlorisActivity.onCreate(FlorisActivity.kt:52)
	at dev.patrickgold.florisboard.settings.ThemeManagerActivity.onCreate(ThemeManagerActivity.kt:124)
	at android.app.Activity.performCreate(Activity.java:6999)
	at android.app.Activity.performCreate(Activity.java:6990)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1214)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2731)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2856)
	at android.app.ActivityThread.-wrap11(Unknown Source:0)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1589)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:164)
	at android.app.ActivityThread.main(ActivityThread.java:6494)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:440)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
Caused by: java.lang.NullPointerException
	at dev.patrickgold.florisboard.ime.core.FlorisBoard$Companion.getInstance(FlorisBoard.kt:182)
	at dev.patrickgold.florisboard.ime.text.gestures.GlideTypingManager$Companion.getInstance(GlideTypingManager.kt:32)
	at dev.patrickgold.florisboard.ime.text.keyboard.KeyboardView.<init>(KeyboardView.kt:80)
	at dev.patrickgold.florisboard.ime.text.keyboard.KeyboardView.<init>(KeyboardView.kt:88)
	... 36 more

**Comment by Glitchy-Tozier** [comment]
Is this v.0.3.10?

**Comment by whispy** [comment]
> Is this v.0.3.10?

Whoops, I forgot to give environment details!

Florisboard v0.3.10
Android 10 running on a Google Pixel XL

**Comment by X-yl** [comment]
Duplicate of #674?

**Comment by patrickgold** [comment]
Thanks for your bug report! This is a duplicate of #674 and I will therefore close it. The latest beta contains a fix for this, the stable channel will receive it once the layout engine rewrite is complete.
